hangul/hangulkeyboard.c

```diff
@@ -815,6 +823,53 @@ hangul_keyboard_list_load_dir(const char* path)
 
     globfree(&result);
     free(pattern);
+#elif _WIN32
+    WIN32_FIND_DATA FindFileData;
+    HANDLE hFind;
+    int n = strlen(pattern) * sizeof(TCHAR) + 1;
```

Comment:
```suggestion
    int n = (strlen(pattern) + 1) * sizeof(TCHAR);
```
널문자도 TCHAR 사이즈로 더해야 할 것 같습니다.